deployments/s5/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "region" {
+  description = "Region"
+  type = string
+  default = "eu-central-1"
+}
+
+variable "env_name" {
+  description = "Environment name"
+  type = string
+}
+
+variable "domain_name" {
```

Comment:
As in S1 / S3, the actual application will run on a subdomain of the root domain name. This may need changing to reflect that (as the CF zone is the root domain name)